x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/field_types/numeric_type.tsx

```diff
@@ -48,28 +48,26 @@ interface Props {
 }
 
 export const NumericType = ({ field, kibanaVersion }: Props) => {
+  const [formData] = useFormData({ watch: 'subType' });
+
   return (
     <>
       <BasicParametersSection>
         {/* scaling_factor */}
-        <FormDataProvider<{ subType?: ComboBoxOption[] }> pathsToWatch="subType">
```

Comment:
The `FormDataProvider` has been deprecated. See: https://github.com/elastic/kibana/blob/main/src/plugins/es_ui_shared/static/forms/hook_form_lib/components/form_data_provider.ts#L34